### ISSUE TITLE
"Set world spawn point" mod prevents minecraft from launching

### ISSUE THREAD

**Issue by HOLYSPAGETI** [ricksouth/serilum-mc-mods#1447]
## **Information**
Minecraft version: 1.18.2
Modloader: Forge 
Forge version: 40.1.70  
Environment: Before the red Mojang loading screen

Mod name: Set world spawn point
Mod version: 1.18.2-2.7

## **Description**
When installed it gives me "Error code 1" preventing the game from launching, no crash reports are produced. I'm using a custom mod pack I'm making made of 238 mods. If it is of some help I can give you the mod pack.

Hereunder the mod list:

[MOD LIST.txt](https://github.com/ricksouth/serilum-mc-mods/files/10147240/MOD.LIST.txt)

**Comment by James103** [comment]
Do you have the latest.log? If so, can you please post it?

**Comment by ricksouth** [comment]
I'll need the logs to see what the problem is. If you use the CurseForge app, can you get them via these steps?

![Screenshot 2022-12-04 214751](https://user-images.githubusercontent.com/38502394/205514738-2f5211f8-fbdf-446c-a436-0b30d53b9d5a.png)
(but not Discord, in here)

**Comment by HOLYSPAGETI** [comment]
Thank you for the quick response. Here you are :) [latest.log](https://github.com/ricksouth/serilum-mc-mods/files/10155085/latest.log)

**Comment by ricksouth** [comment]
Thanks! Does this log crash to "Error code 1"? And is it fixed when removing Set World Spawn Point?

**Comment by HOLYSPAGETI** [comment]
Yes and yes. To create this new latest.log I re-launched the original profile that had the issue at first. After realizing this mod was causing the issue I forked the modpack and never used this version again.
